public/static/js/projects.js

```diff
@@ -304,6 +310,7 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location,
           setup_github_hook: $scope.githubHookID != 0,
           tracks_push_events: data.ProjectRef.tracks_push_events || false,
           pr_testing_enabled: data.ProjectRef.pr_testing_enabled || false,
+          commit_queue: data.ProjectRef.commit_queue || {},
```

Comment:
Initialize to the default values for merge method, action, and status so that the select controls don't have a blank value

Reply:
done